### PR TITLE
Migrations community lab

### PR DESCRIPTION
**Pull request by frjo**
The migration for community labs seems to work well with the exception for the date field.

Could you show my how to get it right?

### CODE REVIEW

File: opentech/apply/funds/management/commands/migrate_community_lab_applications.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    "field_application_mail": {
+        "id": "email",
+        "type": "value",
+        "key": "email",
+    },
+    "field_application_event_date": {
+        "id": "date",
+        "type": "value",
+        "key": "value",
+    },
```
Comment (frjo) [inline opentech/apply/funds/management/commands/migrate_community_lab_applications.py]: I believe the problem is here.

File: opentech/apply/funds/management/commands/seed_community_lab_application.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        self.create_community_lab_round(fund)
+
+    def create_community_lab_form(self):
+
+        data = [
+            {"type": "text_markup", "value": "<h3>Basic information</h3>", "id": "33d38e10-43e3-4a61-8a10-2d3ad760861b"},
+            {"type": "title", "value": {"field_label": "What is your project name?", "help_text": "", "info": None}, "id": "4507372e-a284-488a-ac3c-53b689849d67"},
+            {"type": "full_name", "value": {"field_label": "Your name", "help_text": "", "info": None}, "id": "b62f1c35-52c7-44b5-b7cd-2f1aac6701bd"},
+            {"type": "email", "value": {"field_label": "E-mail", "help_text": "", "info": None}, "id": "02a6cd0e-7ff6-46cd-9bdb-bf5858fe2514"},
+            {"type": "date", "value": {"field_label": "Date of event", "help_text": "", "required": "false", "default_value": ""}, "id": "8c74af9c-6cc7-4558-9d72-0f2c9a87f22b"},
```
Comment (frjo) [inline opentech/apply/funds/management/commands/seed_community_lab_application.py]: This seems to create a date field in proper order. But I get the same issue when filling in an application form manually.

**Comment by zerolab** [comment]
Hi @frjo,

It would help to have as much info as possible. e.g. full stacktrace

**Comment by frjo** [comment]
I closed this since I figured it out. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request closed without merging**